docs/tensor_matrix.rst

```diff
@@ -1,41 +1,41 @@
 .. _tensor:
 
-Tensors and matrices
-====================
+Fields and matrices
+===================
 
-Tensors are global variables provided by Taichi. Tensors can be either sparse or dense.
-An element of a tensor can be either a scalar or a vector/matrix.
+Fields are global variables provided by Taichi. Fields can be either sparse or dense.
+An element of a field can be either a scalar or a vector/matrix.
 
 .. note::
 
-    Although mathematically matrices are treated as 2D tensors, in Taichi, **tensor** and **matrix** are two completely different concepts.
-    Matrices can be used as tensor elements, so you can have tensors with each element being a matrix.
+    Although mathematically matrices are treated as 2D fields, in Taichi, **field** and **matrix** are two completely different concepts.
+    Matrices can be used as field elements, so you can have fields with each element being a matrix.
 
-Tensors of scalars
-------------------
-* Every global variable is an N-dimensional tensor.
+Fields of scalars
+-----------------
+* Every global variable is an N-dimensional field.
 
-  - Global ``scalars`` are treated as 0-D tensors of scalars.
+  - Global ``scalars`` are treated as 0-D fields of scalars.
```

Comment:
```suggestion
  - Global ``scalars`` are treated as 0-D scalar fields.
```
Please update the following changes too, I don't have time to comment them one-by-one.